crates/l2/.env.example

```diff
@@ -11,7 +11,6 @@ DEPLOYER_PRIVATE_KEY=0x385c546456b6a603a1cfcaa9ec9494ba4832da08dd6bcf4de9a71e4a0
 # If set to false, the salt will be randomized.
 DEPLOYER_SALT_IS_ZERO=true
 L1_WATCHER_BRIDGE_ADDRESS=0x266ffef34e21a7c4ce2e0e42dc780c2c273ca440
-L1_WATCHER_TOPICS=0x6f65d68a35457dd88c1f8641be5da191aa122bc76de22ab0789dcc71929d7d37
```

Comment:
Why is this line deleted?

Reply:
Because if we change the event we will have to recalculate the hash again, i've replaced the envar with some code that does this for us:
```rust
let topic = keccak(b"DepositInitiated(uint256,address,uint256,bytes32)");
```